server/misc/constants.js

```diff
@@ -0,0 +1,8 @@
+export const TILE_MAP = {
+  1: 'Gus',
+  2: 'Girder',
+  3: 'Tool',
+  4: 'RedBrickBlock',
+  5: 'BlackBrickBlock',
+  6: 'BreakBrickBlock',
+}
```

Comment:
Ooohhh ok I got these totally wrong, we probably should have gone over them. I can switch them out real quick on my end